format/spec.md

```diff
@@ -940,9 +946,7 @@ Note that partition data tuple's schema is based on the partition spec output us
 The unified partition type is a struct containing all fields that have ever been a part of any spec in the table 
 and sorted by the field ids in ascending order.  
 In other words, the struct fields represent a union of all known partition fields sorted in ascending order by the field ids.
-
 For example,
-
```

Comment:
Can you revert these newline removals? This will cause the following list to not render correctly and will make this sentence part of the preceding paragraph.

Reply:
Whoops , removed.